Example/ViewController.swift

```diff
@@ -52,17 +53,17 @@ class ViewController: UIViewController {
 
     // MARK: Directions Request Handlers
 
-    fileprivate lazy var defaultSuccess: RouteRequestSuccess = { [weak self] (routes) in
-        guard let current = routes.first else { return }
+    fileprivate lazy var defaultSuccess: RouteRequestSuccess = { [weak self] (response) in
+        guard let current = response.routes?.first, case let .route(options) = response.options else { return }
```

Comment:
`current` is unused. Is it just a copy-paste from `resonse.didSet`, and you just meant to check `!response.routes.isEmpty`?